build/SharedFxInstaller.targets

```diff
@@ -254,13 +263,51 @@
       <DebSharedFxProps>$(DebSharedFxProps);DebDependencies=$(DebSharedFxDependencies);SharedFxArchive=$(SharedFxIntermediateArchiveFilePrefix)-linux-x64.tar.gz</DebSharedFxProps>
     </PropertyGroup>
 
-    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDeb" Properties="$(CommonProps);$(DebSharedFxProps)" />
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDeb" Properties="$(CommonProps);$(DebSharedFxProps)" >
+      <Output TaskParameter="TargetOutputs" ItemName="DebsGenerated" />
+    </MSBuild>
 
     <!-- Remove Docker Image to save disk space -->
     <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_RemoveDockerImage" Properties="Image=$(Image)" />
   </Target>
 
   <Target Name="GenerateDebs" DependsOnTargets="_EnsureInstallerPrerequisites">
-    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="PackageVersion=$(PackageVersion);Image=ubuntu.14.04" />
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="PackageVersion=$(PackageVersion);Image=ubuntu.14.04" >
+      <Output TaskParameter="TargetOutputs" ItemName="DebsGenerated" />
+    </MSBuild>
+  </Target>
+
+  <Target Name="_PublishDeb">
+    <PropertyGroup>
+      <DebianRepoServer>azure-apt-cat.cloudapp.net</DebianRepoServer>
+    </PropertyGroup>
+    <ItemGroup>
+      <!-- ubuntu1604x64 -->
+      <DebianRepoId Include="575f40f3797ef7280505232f" />
```

Comment:
Just curious, where does this ID come from? How can we find the IDs for new distros, such as Ubuntu 18?

Reply:
I got these from @wli3 in an email. I assume when a new repo is created for Ubuntu18 then we'll need to update our build configuration to include the new ids.